airflow/models/taskinstance.py

```diff
@@ -105,6 +107,29 @@ def set_current_context(context: Context):
             )
 
 
+def load_error_file(fd: IO[bytes]) -> Optional[Union[str, Exception]]:
+    """Load and return error from error file"""
+    fd.seek(0, 0)
```

Comment:
```suggestion
    fd.seek(0, os.SEEK_SET)
```
(Not important if we don't do this)

Reply:
this is important, no magic number allowed :)